src/officehours_ui/static/css/main.css

```diff
@@ -181,11 +181,11 @@ a:hover {
 	color: #0060ba;
 }
 
-.alert a {
+.alert a:not(.btn) {
 	text-decoration: underline;
 }
 
-p a {
+p a:not(.btn) {
```

Comment:
Because I did not previously add these `:not` lines, I noticed that I had inadvertently added an underline to the "Return to Previous Queue" Button text on the home page, since it was inside an alert.